packages/server/lib/controllers/sync.controller.ts

```diff
@@ -206,6 +207,11 @@ class SyncController {
                 return;
             }
 
+            if (isHosted) {
+                res.send([]);
+                return;
+            }
+
```

Comment:
would it make more sense to short-circuit the `getSyncs` function below or even `syncService`. Otherwise any other usage of those functions requiring a temporal client instance would lead to the same problem again.

Reply:
We need better handling on this in general . This was a temp fix to unblock the community issue